dwave/cloud/utils.py

```diff
@@ -177,6 +179,16 @@ def utcnow():
     return datetime.utcnow().replace(tzinfo=UTC)
 
 
+def epochnow():
+    """Returns now as UNIX timestamp.
+
+    Invariant:
+        epochnow() ~= datetime_to_timestamp(utcnow())
+
+    """
```

Comment:
Oh, I love a good nitpicking session. :)

So, generally, when I say _datetime_, I mean `datetime.datetime`-like object. And when I say _timestamp_, I mean UNIX/POSIX timestamp (seconds since UNIX epoch) - an `int`.

This function is consistent with that. If I've used these nouns in a different context, that's wrong, and I would like to fix it (if you can find any).

By _invariant_ I mean "never changing", i.e. property that remains constant while parameters are changing (or when transformation of time passing is applied to the relation). In other words, no matter what's the current time (_now_), the equation should be satisfied. I didn't use strict equality because `utcnow` will probably run a few microseconds after `epochnow`.